### ISSUE TITLE
Update required packages

### ISSUE THREAD

**Issue by zabbal** [advancedtelematic/ats-garage-docs#81]
* git-core is called git nowadays
* apt is the recommended way to install packages instead of apt-get
* python(3)-devel is missing from required packages

**Comment by pattivacek** [comment]
I agree about all three issues, but to be fair, the reason for this is because the list came almost directly from an older version of Yocto that is still our default, and we generally want to keep it pretty close to their recommendation. However, we've already added some items to their list, and I see no reason why we shouldn't follow your suggestions. Want to make a PR?

**Comment by zabbal** [comment]
Sure, will do once I sort out building the docs with jekyll.
